### ISSUE TITLE
Clarification on fine-tuned weights, training data, and configurations

### ISSUE THREAD

**Issue by SeunggeunKimkr** [KeatingLab/PottsMPNN#1]
Hello, thanks for sharing the PottsMPNN codebase!
Could you provide more details regarding the fine-tuned weights included in this repository? Specifically:
- **Fine-Tuning Process**: Is the weight obtained via supervised fine-tuning using explicit experimental data/stability labels (e.g., $\Delta G$ or $\Delta\Delta G$), or is it self-supervised fine-tuning/further pre-training starting from the base ProteinMPNN weights?
- **Training Dataset**: If supervised fine-tuning was used, what specific datasets and stability labels were used for training?
- **Training Configuration**: Could you share the training configuration, hyperparameters, or scripts used during the fine-tuning process?Thank you!

**Comment by FosterBirnbaum** [comment]
- **Fine-Tuning Process**: The model (the vanilla MSA 0.2 &Aring; noise version) was fine-tuned on the experimental single substitution stability data from the Megascale dataset.
- **Training Dataset**: We used the train/val/test split of the Megascale dataset that was introduced in ThermoMPNN (https://github.com/Kuhlman-Lab/ThermoMPNN).
- **Training Configuration**: I'll update the repo soon with the fine-tuning training scripts.
